js/gemini.js

```diff
@@ -230,9 +230,6 @@ module.exports = class gemini extends Exchange {
         }
         let price = this.safeFloat (order, 'price');
         let average = this.safeFloat (order, 'avg_execution_price');
-        if (average !== 0.0) {
-            price = average; // prefer filling (execution) price over the submitted price
-        }
```

Comment:
If you can split this into separate PRs, I could merge some of these changes rather quickly ) Thx! )

Reply:
It's difficult for me to managed multiple PR's right now but I could do it when I'm done

Reply:
@npomfret 

> when I'm done

that would be awesome! )

thx again!

Reply:
git checkout branch file )